fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/IcebergExternalTable.java

```diff
@@ -274,4 +287,41 @@ public List<SysTable> getSupportedSysTables() {
         makeSureInitialized();
         return SupportedSysTables.ICEBERG_SUPPORTED_SYS_TABLES;
     }
+
+    @Override
+    public boolean isTable() {
+        return catalog.tableExist(null, getRemoteDbName(), getRemoteName());
```

Comment:
Do we need to visit catalog service to check if this is a table or view?
I think we can cache this info, such as a `isView` field in this class and only visit catalog service once?
Same suggestion for `isView()` method.
And I think these operation can be done when initializing this object.